.env

```diff
@@ -0,0 +1,3 @@
+# This env file should be checked into source control
```

Comment:
It would be great to put `DATABASE_URL` here, so everyone can run the app locally without the need to do additional configuration.